.github/workflows/lint.yaml

```diff
@@ -16,7 +16,7 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  GO_VER: '1.21'
+  GO_VER: '1.22'
```

Comment:
maybe 1.22.3

Reply:
Why would you want to restrict the Go version to a specific patch release and not allow the most recent patch release to be selected?

Reply:
If I read the documentation for go correctly, that's the way it's supposed to be now.

https://tip.golang.org/doc/go1.21

`
Go 1.21 introduces a small change to the numbering of releases. In the past, we used Go 1.N to refer to both the overall Go language version and release family as well as the first release in that family. Starting in Go 1.21, the first release is now Go 1.N.0. Today we are releasing both the Go 1.21 language and its initial implementation, the Go 1.21.0 release. These notes refer to “Go 1.21”; tools like go version will report “go1.21.0” (until you upgrade to Go 1.21.1). See “[Go versions](https://tip.golang.org/doc/toolchain#version)” in the “Go Toolchains” documentation for details about the new version numbering.
`

Reply:
This value is being to configure the setup-go action: https://github.com/actions/setup-go

Reply:
ok. Then don't.